source/draw/README.md

```diff
@@ -17,6 +17,11 @@ The destination is similar to the viewport by defining where to draw. The differ
 ## Transform
 Transform is the inverse camera matrix applied after the destination scaling.
 
+## FlatTransform
+FlatTransform is applied after the projection in 2D.
+It works without parallax by tilting the Z axis in the projection matrix.
+This is done by multiplying the projection matrix with a normalized translation which will skew instead of translate the projection matrix.
```

Comment:
I had to add this only to make sure that no one takes back the viewport hack believing that the parallax problem still exists.